Dev-Corner/slides/tool_integration.html

```diff
@@ -1100,6 +1270,22 @@
 
                     <section data-markdown>
                         <script type="text/template">
+                            <img src="../../shared/images/bioconda_logo.png" class="no-border"  height=50 />
+
+                            1. Clone `bioconda/bioconda-recipes.git` or a fork
+                            2. Create a new branch "package_new_login"
+                            3. Fill two files `meta.yaml` and `build.sh`
+                            4. Build your new package and test it using `conda build`
```

Comment:
It would be better to use the simulate-travis.py script, but it means we need a miniconda3 install (xref https://github.com/bioconda/bioconda-recipes/issues/3275 https://github.com/galaxyproject/planemo/issues/604) which is not yet the case with planemo conda_init. Don't know if it's doable to fix conda_init before the strasbourg affair though